website/templates/util/render_node.mako

```diff
@@ -51,16 +51,26 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
```

Comment:
It is always false for non-registration nodes, but if it is a registration it can be true while it is archiving.